app/views/writable/_editor.haml

```diff
@@ -1,50 +1,50 @@
--# locals: ( item: )
+-# locals: ( writable: )
```

Comment:
I support this move and variable renaming tbh

Reply:
Possibly it should be pulled out into another PR tho?

Reply:
I could do that if you think this PR will still need a lot of work before merging?

Reply:
Dunno! It's kind of terrifying but it does seem like it has solid tests.